src/components/KYCWall/BaseKYCWall.js

```diff
@@ -61,14 +63,14 @@ class KYCWall extends React.Component {
     getAnchorPosition(domRect) {
         if (this.props.popoverPlacement === 'bottom') {
             return {
-                anchorPositionTop: domRect.top + (domRect.height - 2),
-                anchorPositionLeft: domRect.left + 20,
+                anchorPositionVertical: domRect.top + (domRect.height - 2),
+                anchorPositionHorizontal: domRect.left + 20,
             };
         }
 
         return {
-            anchorPositionTop: domRect.top - 150,
-            anchorPositionLeft: domRect.left,
+            anchorPositionVertical: domRect.top - 150,
+            anchorPositionHorizontal: domRect.left,
         };
```

Comment:
This change caused a regression [here](https://github.com/Expensify/App/issues/21259). `anchorPositionVertical` value was not set correctly here after moving from `anchorPositionTop`, we updated it correctly [here](https://github.com/Expensify/App/pull/21980/files)